scalafmt-tests/src/test/resources/newlines/source_unfold.stat

```diff
@@ -4350,7 +4437,7 @@ enum Namespace(val uri: String | Null):
 package dev.muve.vdom
 
 enum Namespace
-  (val uri: String | Null):
+    (val uri: String | Null):
```

Comment:
```suggestion
  (val uri: String | Null):
```
we already have an indentation of two here and we don't set the extra indentation. Why did this change?

Reply:
So it's `indent.significant + extra`? I would say 0 is a better default. It might be hard for people to figure out where they are getting the additional 2

Reply:
will fix shortly

Reply:
thanks!

Reply:
done, ptal